docs/running_on_testnet.md

```diff
@@ -88,13 +88,14 @@ Currency: DAT
 ```
 Note you can only use the faucet once per day. Use the testnet faucet available at https://faucet.vana.org to fund your wallets, or ask a DAT holder to send you some test DAT tokens.
 
-Get your wallet address for all three accounts
+Get your wallet private keys for all three accounts and import them into metamask or another wallet of your choice.
```

Comment:
Maybe I'm missing it, but what does importing them into metamask help with?

Reply:
Sending DLPT from owner address to validators is not possible via CLI tool currently

Reply:
You don't need to import a wallet to send tokens to it though?

Reply:
You need to import owner wallet, validators not neccessary, but I think thats good to have them for visibility